invenio_accounts/models.py

```diff
@@ -564,12 +561,15 @@ class Domain(db.Model, Timestamp):
     org_id = db.Column(db.Integer(), db.ForeignKey(DomainOrg.id), nullable=True)
     """Organisation associated with domain."""
 
-    org = db.relationship("DomainOrg", back_populates="domains")
+    org = db.relationship("DomainOrg", backref="domains")
```

Comment:
Since the category can be null I would just leave it blank, and let instances define values themselves.......my guess is most instances won't need it....an admin interface to edit would be needed at some point :-)